src/CollabVM.h

```diff
@@ -129,6 +129,7 @@ class CollabVMServer : public std::enable_shared_from_this<CollabVMServer> {
 	void ExecuteCommandAsync(std::string command);
 	void MuteUser(const std::shared_ptr<CollabVMUser>& user, bool permanent);
 	void UnmuteUser(const std::shared_ptr<CollabVMUser>& user);
+	void OnAutotypeInstruction(const std::shared_ptr<CollabVMUser>& user, std::string clipboard);
```

Comment:
remove (this shouldn't hurt anything, because its `extern "C++"` implicitly, and therefore, the GuacamoleInstruction overload will be used, but the change request I gave would make this dead code that if somehow called would cause linker errors)